src/SongPosition.cpp

```diff
@@ -3,6 +3,10 @@
 
 static Preference<float> g_fVisualDelaySeconds( "VisualDelaySeconds", 0.0f );
 
+SongPosition::SongPosition() {
```

Comment:
Can just keep the constructor in the header file, especially since it's using a function that's now defined in the header.